x-pack/plugins/cloud_security_posture/public/components/cloud_security_data_table/fields_selector/fields_selector_table.tsx

```diff
@@ -56,8 +56,34 @@ export interface FieldsSelectorTableProps {
   onAddColumn: (column: string) => void;
   onRemoveColumn: (column: string) => void;
   title: string;
-  onFilterSelectedChange: (enabled: boolean) => void;
-  isFilterSelectedEnabled: boolean;
+}
+
+function filterFieldsBySearch(
+  dataView: DataView,
+  columns: string[] = [],
+  searchQuery?: string,
+  isFilterSelectedEnabled: boolean = false
+) {
+  const dataViewFields = dataView.fields
```

Comment:
quite a lot of logic here, does it make sense to add some unit tests specifically for this function?

Reply:
Yes I agree,  I was trying to get this coverage while rendering the component in the tests but it does not seem to be able to "react" to the change in the searchbox.

I've exported it and now testing in fields_selector_table.test.tsx